bedrock/mozorg/templates/mozorg/about/forums/forums.html

```diff
@@ -37,20 +37,21 @@ <h1><span class="highlight">{{ self.page_title() }}</span></h1>
   <a href="https://www.giganews.com/"><img src="{{ static('img/mozorg/about/giganews.png') }}" alt="" height="25" width="89" /></a>
   </p>
 
-  <form method="get" action="https://groups.google.com/groups">
+  <form method="get" action="https://groups.google.com/groups" class="c-forum-search">
   <a href="https://www.google.com/"><img
        src="https://www.google.com/logos/Logo_25wht.gif"
```

Comment:
Not new to this PR, but fwiw this image will not show up in production as it violates our CSP (see https://github.com/mozilla/bedrock/issues/6684)